server/src/ClientToDBAPI/UsersRoute/mainRoute.ts

```diff
@@ -140,6 +140,25 @@ usersRoute.post('/updateIsUserActiveById', adminMiddleWare, (request: Request, r
     updateIsUserActive(response, request.body.userId, request.body.isActive);
 })
 
+usersRoute.post('/deactivateAllCountyUsers', (request: Request, response: Response) => {
+    const deactivateAllCountyUsers = logger.setup({
+        workflow: 'updating county users activity status to false',
+        user: response.locals.user.id,
+    });    
+    const parameters = { countyId: request.body.countyId }
```

Comment:
change to request.body.county (for the future middleware)

Reply:
changed that